package.json

```diff
@@ -44,6 +44,7 @@
     "express": "^4.17.1"
   },
   "devDependencies": {
+    "@mswjs/interceptors": "^0.17.3",
```

Comment:
Oh, I see the pickle here: since it's a transient dependency, it needs to be bumped from the MSW's side.

Can you try removing this dependency and doing `yarn install`? That should pull in the latest interceptors as MSW depends on a range.

Reply:
done, I've cleaned up commits